Week2/todo_app/program.js

```diff
@@ -22,51 +22,151 @@ class TodoModel {
     }
 
     create(description, callback) {
-        // Write code and query to create a new TODO item
+        const createQuery = `
+        INSERT INTO todo_items(text , is_completed , user_id)
+        VALUES (${dbConnection.escape(description)} , 0 ,${dbConnection.escape(args2)} )`;
```

Comment:
:+1: 

Good job using escaping!

Reply:
Your use of commas is a bit weird, take a quick look at that. `(text , is_completed , user_id)` should be `(text, is_completed, user_id)`.